relay-replays/src/recording/protocol.rs

```diff
@@ -0,0 +1,171 @@
+/// Protocol.
+///
+/// The following section includes the behaviors responsible for deserializing a recording from
+/// its HTTP transport schema and serializing a recording to its Kafka transport schema.
+///
+/// We expect recordings to come in the format of plaintext headers (JSON encoded), then a new
+/// line character, then an optionally compressed RRWeb recording.  Failure to include any
+/// component of this schema results in an error.
+use flate2::{read::ZlibDecoder, write::ZlibEncoder, Compression};
+use serde::Deserialize;
+use serde_json::Error as SError;
+use std::fmt::Display;
+use std::io::{Error, Read, Write};
```

Comment:
Please reorder imports:

```suggestion
use std::fmt::Display;
use std::io::{Error, Read, Write};

use flate2::{read::ZlibDecoder, write::ZlibEncoder, Compression};
use serde::Deserialize;
use serde_json::Error as SError;
```